rosdep/base.yaml

```diff
@@ -4461,6 +4461,13 @@ qtbase5-dev:
   rhel: [qt5-qtbase-devel]
   slackware: [qt5]
   ubuntu: [qtbase5-dev]
+qtdeclarative5-controls-plugin:
+  arch: [qt5-quickcontrols]
+  debian: [qtdeclarative5-controls-plugin]
```

Comment:
I can't find this debian package: https://packages.debian.org/search?suite=default&section=all&arch=any&searchon=names&keywords=qtdeclarative5